docs/services/data/_index.md

```diff
@@ -108,28 +114,17 @@ Upload data to the Viam app from your local computer or mobile device using the
 {{% /manualcard %}}
 {{< /cards >}}
 
-#### Used with
+### Query your data
 
-You can configure the frequency of data capture individually for each supported component:
+Once you have [synced](/services/data/capture-sync/), you can query the data you've collected in multiple ways, including through the [data client API](/appendix/apis/data-client/) or [inside the Viam app](/how-tos/sensor-data-query-with-third-party-tools/).
+For _tabular_ sensor data, you can [run {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} queries](/how-tos/sensor-data-query-sdk/) against your synced data from the [Query subtab](https://app.viam.com/data/query) of the **Data** tab in the Viam app.
 
 {{< cards >}}
-{{< relatedcard link="/components/arm/">}}
-{{< relatedcard link="/components/board/">}}
-{{< relatedcard link="/components/camera/">}}
-{{< relatedcard link="/components/encoder/">}}
-{{< relatedcard link="/components/gantry/">}}
-{{< relatedcard link="/components/motor/">}}
-{{< relatedcard link="/components/movement-sensor/">}}
-{{< relatedcard link="/components/sensor/">}}
-{{< relatedcard link="/components/servo/">}}
+{{% card link="/how-tos/sensor-data-query-with-third-party-tools/" %}}
+{{% card link="/how-tos/sensor-data-query-sdk/" %}}
 {{< /cards >}}
 
-## Query your data
-
-Once you have [synced](/services/data/cloud-sync/), you can query the data you've collected in multiple ways, including through the [data client API](/appendix/apis/data-client/) or [inside the Viam app](/how-tos/sensor-data-query-with-third-party-tools/).
-For _tabular_ sensor data, you can [run {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} queries](/how-tos/sensor-data-query-sdk/) against your synced data from the [Query subtab](https://app.viam.com/data/query) of the **Data** tab in the Viam app.
-
-## Permissions
+### Permissions
 
 Data management permissions vary between owners and operators.
 For more information about who can do what with data, see [Data Permissions](/cloud/rbac/#data-and-machine-learning).
```

Comment:
The API that you have on this page should be on the Data Capture and Sync page because it is the API of the data management service. On this page we probably want to instead have the API tables from https://docs-test.viam.dev/3473/appendix/apis/data-client/#api. And we may also want to link to https://docs-test.viam.dev/3473/cli/#data